addons/gdUnit3/src/cmd/CmdOption.gd

```diff
@@ -0,0 +1,46 @@
+class_name CmdOption
+extends Reference
+
+
+var _commands :Array
+var _help :String
+var _description :String
+var _type :int
+var _arg_optional :bool = false
+
+
+func _init(commands :String, help :String, description :String, type :int = TYPE_NIL, arg_optional :bool = false):
```

Comment:
add doc e.g. `commands` is a coma separated list of commands, beginning with the short form